ArduPlane/tailsitter.cpp

```diff
@@ -198,13 +206,21 @@ void Tailsitter::output(void)
         if (hal.util->get_soft_armed() && in_vtol_transition() && !quadplane.throttle_wait && quadplane.is_flying()) {
             /*
               during transitions to vtol mode set the throttle to
-              hover thrust, center the rudder and set the altitude controller
+              hover thrust or Q_TAILSIT_THR_VT, center the rudder and set the altitude controller
               integrator to the same throttle level
               convert the hover throttle to the same output that would result if used via AP_Motors
               apply expo, battery scaling and SPIN min/max. 
             */
-            throttle = motors->thrust_to_actuator(motors->get_throttle_hover()) * 100;
-            throttle = MAX(throttle,plane.aparm.throttle_cruise.get());
+            if (!is_negative(transition_throttle_vtol)) { 
+                // Q_TAILSIT_THR_VT is positive use it until transition is complete
+                throttle = MIN(transition_throttle_vtol,100.0);
+                throttle = motors->thrust_to_actuator(throttle);
```

Comment:
I don't think we need this. Firstly it is expecting a 0-1 value and secondly this applies throttle expo, I think that just makes it less clear to the user that it is doing the throttle they asked for. Not that fussed if you think hving the expo is better in that case it should be:

`throttle = motors->thrust_to_actuator(throttle * 0.01) * 100;`

Reply:
if its good for hover throttle, why not a user value? battery comp,etc...

Reply:
Fair point about battery comp, maybe it is better to have the corrected value.